evil-collection-docker-container.el

```diff
@@ -0,0 +1,50 @@
+;;; evil-collection-docker-container.el --- Evil bindings for docker-container -*- lexical-binding: t -*-
+
+;; Package-Requires: ((emacs "25.1"))
+;; Keywords: evil, docker-container, tools
+
+;; This program is free software; you can redistribute it and/or modify
+;; it under the terms of the GNU General Public License as published by
+;; the Free Software Foundation, either version 3 of the License, or
+;; (at your option) any later version.
+
+;; This program is distributed in the hope that it will be useful,
+;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;; GNU General Public License for more details.
+
+;; You should have received a copy of the GNU General Public License
+;; along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+;;; Commentary:
+;; Evil bindings for docker-container.
+
+;;; Code:
+(require 'docker-container)
+(require 'evil-collection)
+
+(defconst evil-collection-docker-container-maps '(docker-container-mode-map))
+
+(defun evil-collection-docker-container-setup ()
+  "Set up `evil' bindings for `docker-container'."
+  (evil-collection-define-key 'normal 'docker-container-mode-map
+    "q" 'quit-window
+    "?" 'docker-container-help-popup
+    "C" 'docker-container-cp-popup
+    "D" 'docker-container-rm-popup
+    "I" 'docker-container-inspect-popup
+    "K" 'docker-container-kill-popup
+    "L" 'docker-container-logs-popup
+    "O" 'docker-container-stop-popup
+    "P" 'docker-container-pause-popup
+    "R" 'docker-container-restart-popup
+    "S" 'docker-container-start-popup
+    "a" 'docker-container-attach-popup
+    "b" 'docker-container-shell-popup
+    "d" 'docker-container-diff-popup
+    "f" 'docker-container-find-file-popup
+    "l" 'docker-container-ls-popup
```

Comment:
This can be an unexpected binding. I think it's better to avoid binding h/j/k/l/ to non-movement commands.

Reply:
@wedens: right. Will try to find something else :wink: